flink-connectors/flink-connector-hbase-base/src/test/java/org/apache/flink/connector/hbase/util/HBaseSerdeTest.java

```diff
@@ -71,16 +69,15 @@ public void convertToNewRowTest() {
         }
 
         // this verifies RowData is not reused
-        assertFalse(resultRowDatas.get(0) == resultRowDatas.get(1));
-
-        List<String> expected = new ArrayList<>();
-        expected.add("+I(1,+I(10),+I(Hello-1,100),+I(1.01,false,Welt-1))");
-        expected.add("+I(2,+I(20),+I(Hello-2,200),+I(2.02,true,Welt-2))");
-        assertEquals(expected, resultRowDataStr);
+        assertThat(resultRowDatas.get(0)).isNotSameAs(resultRowDatas.get(1));
```

Comment:
nit: I'm wondering whether we could make this part of the assert using the `as()` method. The documentation would be present still but also included in the assertion error output...

Reply:
same applies in the `convertToReusedRowTest` test (line 92)